### DISCUSSION TITLE
Output on Union Type with Enum

### DISCUSSION THREAD

**Discussion by sabinadams** [jasonkuhrt/molt#146] [Q&A]
I have Command set up as follows:

```ts
const args = Command.parameters({
  "t template": z.string(),
  "i install": z
    .boolean()
    .or(z.enum(["npm", "yarn", "pnpm"]))
    .default(false),
  "n name": z.string(),
  "p path": z.string(),
}).parse();
```

In the outputted docs I was expecting the **Type** field of the `install` argument to show an output similar to the one described here which describes the types in the union as well as the individual options in the enum:
https://github.com/jasonkuhrt/molt/issues/107#issuecomment-1366319040

Probably not a bug per-se, but is the output I receive expected? 
#### Screenshot
<img width="1512" alt="CleanShot 2023-03-07 at 13 44 48@2x" src="https://user-images.githubusercontent.com/18456526/223440387-a1c47794-a58e-4d5d-ade1-61bced1e1045.png">

**Comment by jasonkuhrt** [comment]
@sabinadams Yep there's some info about this rendering here: https://github.com/jasonkuhrt/molt/tree/main/packages/@molt/command#union. 

I will add a setting to explicitly control the rendering.

**Reply by sabinadams** [reply]
Thanks! That helped. 

As of this [PR](https://github.com/prisma/try-prisma/pull/38) in try-prisma, Molt has replaced Commander.

**Reply by jasonkuhrt** [reply]
Amazing! I think this upcoming feature might be useful to you there https://github.com/jasonkuhrt/molt/issues/9 Let me know if there are any requirements and/or nice-to-haves that you'd want that are missing there!

**Comment by jasonkuhrt** [comment]
@sabinadams I've shipped a feature now that allows you to _force_ the expanded style _even if there aren't descriptions for the parameters_. While in general descriptions are probably a "you should do that" kind of thing (for the CLI author) there's now more flexibility around this behaviour as well.

Doc details about this are here https://github.com/jasonkuhrt/molt/tree/main/packages/@molt/command#help-rendering.
